src/test/ui/rfc-2565-param-attrs/param-attrs-pass.rs

```diff
@@ -0,0 +1,21 @@
+// compile-pass
+
```

Comment:
Please also add checks that `#[allow/warn/deny/forbid(...)]` works properly. E.g. `#[allow(unused_variables)]` is a good candidate, I think.

When doing so, also account for `cfg_attr` here semantically (and use revisions for that).

Reply:
Thanks for reminding about `#[allow/warn/deny/forbid]`